packages/core/src/components/AccordionItem.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { Pressable, StyleSheet, View } from "react-native";
+
+import Text from "./Text";
+import type { IconSlot } from "../interfaces/Icon";
+import {
+  COMPONENT_TYPES,
+  createIconProp,
+  createTextProp,
+} from "@draftbit/types";
+
+type Props = {
+  icon?: string;
+  title: string;
+} & IconSlot;
+
+const AccordionItem = ({ Icon, icon, title }: Props) => {
```

Comment:
```suggestion
const AccordionItem = ({ Icon, icon, title, style }: Props) => {
```

You can also do this:
https://github.com/draftbit/react-native-jigsaw/blob/a421be23d58d60b71257a16eb8db11d13a4713bb/packages/core/src/components/FormRow.tsx#L25

```
  const { textStyles, viewStyles } = extractStyles(style);
```

this will separate the textStyles and viewStyles so you can apply viewStyles to pressable and textStyles to the title

Reply:
done